terraform/variables.tf

```diff
@@ -173,3 +179,14 @@ variable "distributed_tff_example_worker_2_address" {
   description = "Address of the second worker of the distributed TensorFlow Federated example. Set this when the worker is outside the coordinator mesh."
   type        = string
 }
+
+variable "cross-device" {
+  description = "Enable cross device infrastructure deployment"
+  type        = bool
+  default     = false
+}
+
+variable "spanner_instance_config" {
```

Comment:
Is there a way we can automatically deduce this value based on other inputs? Also, provide a default value in line with the default value we set for the `region` above.

Reply:
It seems that we can leverage on the region. Let me test that.

Reply:
It does not work because Spanner has its own implementation (https://cloud.google.com/spanner/docs/instance-configurations#multi-region-configurations)

Reply:
Can you provide a default value then?